### ISSUE TITLE
Get video dimensions and set aspect ratio based on that

### ISSUE THREAD

**Issue by crearo** [ct7ct7ct7/Android-VimeoPlayer#11]
First of all, thanks for your great work! I was going through the Youtube version of this, and was bummed that I might have to write the Vimeo one. But saw you've already done it, and done it well! Thanks!

I have 2 types of videos; landscape and portrait. I am placing the player views in a recyclerview and want to dynamically resize the element based on the video width and height. Do you know if I can get the video dimensions?

**Comment by ct7ct7ct7** [comment]
Do you mean "get the video resolution"?

Unfortunately, We can't get the Vimeo video quality.

**Comment by crearo** [comment]
Yeah video resolution/dimension. Basically width x height in pixels.
